--- nom-kconfig-0.12.0/Cargo.toml	2006-07-24T01:21:28+00:00
+++ nom-kconfig-0.12.0/Cargo.toml	2026-06-13T18:51:13.008296+00:00
@@ -129,11 +129,6 @@
 name = "source_glob"
 path = "tests/source_glob.rs"
 
-[[bench]]
-name = "my_benchmark"
-path = "benches/my_benchmark.rs"
-harness = false
-
 [dependencies.glob]
 version = "0.3.3"
 optional = true
@@ -158,10 +153,6 @@
 version = "4.4.18"
 features = ["derive"]
 
-[dev-dependencies.criterion]
-version = "0.5.1"
-features = ["html_reports"]
-
 [dev-dependencies.reqwest]
 version = "0.13.4"
 features = ["blocking"]
